docs/setup/new_cluster.md

```diff
@@ -0,0 +1,695 @@
+Setting up a new CommCare HQ environment on a small cluster
+===========================================================
+
+This document will walk you through the process of setting up a new
+cluster of virtual machines to run CommCare HQ.
+
+
+Planning
+--------
+
+Start by estimating the resources that the cluster will need.
+
+The following **example** uses a cluster of almost identically resourced
+VMs. We have estimated that the following will meet the requirements of
+our project:
+
+| Name       | vCPUs | RAM      | Storage                 |
+| ---------- | -----:| --------:|:----------------------- |
+| control1   |     2 | 4096 MiB | 30 GiB                  |
+| proxy1     |     2 | 4096 MiB | 30 GiB                  |
+| webworker1 |     2 | 8192 MiB | 30 GiB                  |
+| webworker2 |     2 | 8192 MiB | 30 GiB                  |
+| db1        |     2 | 8192 MiB | 30 GiB + 60 GiB         |
+| db2        |     2 | 8192 MiB | 30 GiB + 60 GiB + 20 GiB|
+
+db1 has additional storage for databases, and db2 has one extra volume
+for databases, and another for a shared NFS volume.
+
+
+(Example) Creating a small cluster using libvirt with KVM
```

Comment:
I'm not sure we should include this, as we don't support the VM level stuff on the machines (and in fact, we prefer you use some cloud hosting provider to do this for you). If you wanted to keep these docs around for us to reference, I'd stick them in confluence.

Reply:
Confluence might be a good idea. I'd like to keep this around for reference, because setting this up myself was really useful for working through the process we are expecting third-party / on-prem hosting providers to follow. Some projects who want to self-host are doing so explicitly because they don't want to host in the cloud.